src/main/java/banking/account/view/BankView.java

```diff
@@ -0,0 +1,146 @@
+package banking.account.view;
+
+
+import banking.account.controller.BankController;
+import banking.account.domain.BasicAccount;
+import banking.account.dto.AccountDTO;
+import banking.account.exception.FormatException;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.text.DecimalFormat;
+import java.util.Scanner;
+
+import static banking.account.constant.ConstantMessage.*;
+
+public class BankView {
+    private static final Scanner sc = new Scanner(System.in);
+    private BankController bankController = BankController.getInstance();
```

Comment:
```suggestion
    private final BankController bankController = BankController.getInstance();
```
final 키워드를 사용해 변경되지 않을 인스턴스라는 것을 명시해야 할 것 같아요